pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1392,12 +1409,12 @@ func (vs *VSphere) GetZone(ctx context.Context) (cloudprovider.Zone, error) {
 		}
 
 		if zone.Region == "" {
-			if vs.cfg.Labels.Region != "" {
+			if vs.cfg.Labels.Region != DefaultRegionCategoryName {
```

Comment:
GetZone method seems to assume that the k8s cluster is deployed in one single VC. So, zones would not work if the k8s cluster is deployed across multiple VCs? Note that VCP is designed to work when the k8s cluster is deployed across multiple VCs. 

One question - is zones feature optional? If so, how can one run VCP without zones support?

Reply:
Multi-VC has not been tested with this change.

The zones feature is optional to use. When there are no zone tags in vCenter, VCP works as it did before this change.